packages/core/src/axis-chart.ts

```diff
@@ -113,9 +115,36 @@ export class AxisChart extends Chart {
 			},
 		};
 
+		// Add extra legend items
+		const extraLabels = [];
+		const radiusLabel = Tools.getProperty(
+			options,
+			"bubble",
+			"radiusLabel"
+		);
+		if (radiusLabel) {
+			const outerRadius = Tools.getProperty(
+				LegendConfig,
+				"checkbox",
+				"radius"
+			);
+			const innerRadius = outerRadius - 3;
+
+			extraLabels.push({
+				type: "radius-label",
+				icon: `<svg width="16px" height="16px" viewBox="0 0 16 16">
+							<g fill="none" fill-rule="evenodd">
+								<circle cx="7" cy="7" r=${outerRadius}></circle>
+								<circle cx="7" cy="10" r=${innerRadius}></circle>
+							</g>
+						</svg>`,
+				text: radiusLabel
+			});
```

Comment:
So why would this not be done through the charting options? and we can set the default value in bubble charts through `configuration.ts`

Reply:
maybe @jeanservaas should chime in here. What we're wondering is whether only the library should be able to add additional legend items internally, or whether we'd like to also extend that capability to the users of the library?

Reply:
@theiliad my gut is to extend the capability to the users of the library, what do you think?

Reply:
Yes agreed as long as we have clear guidelines and users aren't overusing the feature